src/codegen/property_body.rs

```diff
@@ -76,125 +76,123 @@ impl<'a> Builder<'a> {
     }
 
     fn chunks_for_get(&self) -> Vec<Chunk> {
-        let mut params = Vec::new();
+        if self.is_child_property {
+            let mut body = Vec::new();
+
```

Comment:
Weird coding style. ;)

Reply:
Which parts of it? :) I mean, my code fixes simplifies the properties getters/setters; it doesn't do the same for the GTK 3 child properties

Reply:
Maybe you can move the whole child property generation into a generate_getter/setter function so it doesn't pollute the main code path? Who cares about GTK child properties after all :)